nextcloud-startup-script.sh

```diff
@@ -235,18 +235,20 @@ touch "$SCRIPTS/you-can-not-run-the-startup-script-several-times"
 echo
 print_text_in_color "$ICyan" "Getting scripts from GitHub to be able to run the first setup..."
 # Scripts in static (.sh, .php, .py)
-download_static_script temporary-fix
-download_static_script update
-download_static_script trusted
-download_static_script test_connection
-download_static_script setup_secure_permissions_nextcloud
-download_static_script change_db_pass
-download_static_script nextcloud
-download_static_script update-config
-download_static_script apps
-download_static_script configuration
-download_le_script activate-tls
-download_static_script menu
+download_script LETS_ENC activate-tls
+download_script STATIC update-config
+download_script STATIC temporary-fix
+download_script STATIC update
+download_script STATIC trusted
+download_script STATIC test_connection
+download_script STATIC setup_secure_permissions_nextcloud
+download_script STATIC change_db_pass
+download_script STATIC nextcloud
+download_script STATIC menu
+download_script STATIC server_configuration
+download_script STATIC nextcloud_configuration
```

Comment:
I am against this change because we will break backwards compatibility if we change the order of the variables like I have planned to do.

Why do you want this so madly?

I would only do this after #11 is fixed because it will probably change to be able to fix it...

As additionally the moving of files to another folder will complicate things in the future if we do it this way...

Reply:
This follows the rest of the structure and idea with the startup script - all scripts should be made available to run directly with bash. Your "get latest script" script actually removes the file then re-downloads it again. 

The idea here is to not having to download things **again** if something should happen to internet while while the first setup is run.

Are you following on what I mean?

Another option would be to do it your way and just trust that once out connectivity check is done (first thing that happens when the startup script is run) execute and download whatever as many times as we want.

Reply:
No, that wasn't what I meant.
I would just revert changing to the new function "download_script" because it has the potential to break backwards compatibility in the future.

Reply:
Actually doing what you have proposed makes sense but not with the new function.
I would do it this way:
```
download_le_script activate-tls
download_static_script update-config
download_static_script temporary-fix
download_static_script update
download_static_script trusted
download_static_script test_connection
download_static_script setup_secure_permissions_nextcloud
download_static_script change_db_pass
download_static_script nextcloud
download_static_script menu
download_static_script server_configuration
download_static_script nextcloud_configuration
```

Reply:
I've done that now.